core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html

```diff
@@ -92,7 +92,7 @@
              this font.
             -->
             <span ng-if="isAdmin" class="oppia-navbar-role-text" style="right: 4px; font-size: 11px;">A</span>
-            <span ng-if="isModerator && !isAdmin" class="oppia-navbar-role-text">M</span>
+            <span ng-if="isModerator && !isAdmin" class="oppia-navbar-role-text" style="font-size: 11px;">M</span>
```

Comment:
Instead of inline styling, it would be better to add such styling inside the style tag (Line 235).
Also, remove that inline styling (only font-size) from line 94.

Reply:
Done :+1: